src/importexport/guitarpro/internal/importgtp.cpp

```diff
@@ -84,6 +84,7 @@
 #include "engraving/dom/vibrato.h"
 
 #include "log.h"
+#include "global/thirdparty/utfcpp-3.2.1/utf8.h"
```

Comment:
better use our class  UtfCodec, instead of thirdparty directly 
(add to UtfCodec is_valid method)